.circleci/config.yml

```diff
@@ -1,20 +1,35 @@
 version: 2
 jobs:
-  prepare web dependencies cache:
+  prepare dependencies cache:
     docker:
       - image: node:8
     steps:
       - checkout
       - restore_cache:
-          key: v1-cms-hitech-apd-{{ checksum "web/package-lock.json" }}
+          key: cms-hitech-apd-npm-{{ checksum "web/package-lock.json" }}-{{ checksum "api/package-lock.json" }}
+      - run:
+          name: update npm
+          command: npm install -g npm@6
       - run:
-          name: install dependencies
+          name: install web dependencies
           working_directory: ~/project/web
-          command: npm install
+          command: npm ci
+      - run:
+          name: install api dependencies
+          working_directory: ~/project/api
+          command: npm ci
+      - save_cache:
+          key: cms-hitech-apd-npm-{{ checksum "web/package-lock.json" }}-{{ checksum "api/package-lock.json" }}
+          paths:
+            - /root/.npm
       - save_cache:
           key: v1-cms-hitech-apd-{{ checksum "web/package-lock.json" }}
           paths:
             - web/node_modules
+      - save_cache:
+          key: v1-cms-hitech-api-{{ checksum "api/package-lock.json" }}
+          paths:
+            - api/node_modules
```

Comment:
Instead of separate tasks for installing web and api dependencies, now we have just one dependencies task.  It slows down the build slightly, but we have less duplication in the config file now.  🤷‍♂️